stdlib/hashes/sha256/shaRound.code

```diff
@@ -0,0 +1,13 @@
+import "LIBSNARK/sha256round" as sha256
+
+def main(field[256] a, field[256] b, field[256] IV) -> (field[256]):
+
+  // a and b is NOT checked to be of type bool
```

Comment:
maybe we could call such files `shaRoundUnchecked` to signal they're not binary checked?

Reply:
Good idea. Change to `shaRoundNoBoolCheck.code`